content/en/docs/reference/scheduling/config.md

```diff
@@ -289,7 +291,187 @@ only has one pending pods queue.
 {{% /tab %}}
 {{< /tabs >}}
 
-## {{% heading "whatsnext" %}}
+### Plugins that apply to multiple extension points {#multipoint}
+
+There is an additional field in the profile config, `multiPoint`, which allows 
+for easily enabling or disabling a plugin across several extension points. The intent 
+of `multiPoint` config is to simplify the configuration needed for users and administrators 
+when using custom profiles.
+
+Consider a plugin, `MyPlugin`, which implements the `preScore`, `score`, `preFilter`, 
+and `filter` extension points. To enable `MyPlugin` for all its available extension 
+points, the profile config looks like:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: multipoint-scheduler
+    plugins:
+      multiPoint:
+        enabled:
+        - name: MyPlugin
+```
+
+This would equate to manually enabling `MyPlugin` for all of its extension 
+points, like so:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: non-multipoint-scheduler
+    plugins:
+      preScore:
+        enabled:
+        - name: MyPlugin
+      score:
+        enabled:
+        - name: MyPlugin
+      preFilter:
+        enabled:
+        - name: MyPlugin
+      filter:
+        enabled:
+        - name: MyPlugin
+```
+
+One benefit of using `multiPoint` here is that if `MyPlugin` implements another 
+extension point in the future, the `multiPoint` config will automatically enable it 
+for the new extension.
+
+Specific extension points can be excluded from `MultiPoint` expansion using 
+the `disabled` field. This works with disabling default plugins, out-of-tree plugins, 
```

Comment:
I think you should say non-default plugins. There are official in-tree plugins that are disabled by default, and you can affect those. The out-of-tree plugins behave similarly to custom in-tree plugins.

Reply:
Sounds good, updated it to say non-default